packages/customer/integration-tests/__tests__/services/customer-module/index.spec.ts

```diff
@@ -23,19 +23,308 @@ describe("Customer Module Service", () => {
   })
```

Comment:
comment,nitpicking(this-file): Didn't see this in previous PR, but I think we should stick with one way of testing async code. 

So instead of `expect.resolves`, we await the result and assert on that. Unless there is a benefit to using `resolves`?

Reply:
agreed

Reply:
I prefer this approach, especially when it comes to testing for errors. E.g.:

```javascript
  expect.assertions(1);
  try {
    await doPromiseStuff();
  } catch (error) {
    expect(error).toMatch('bad');
  }
```

vs.

```javascript
  await expect(doPromiseStuff()).rejects.toMatch("bad");
```

Reply:
we usually do
```ts
const err = await doPromiseStuff().catch(e => e)
```

Reply:
Since this is syntactic sugar, I would say we should follow the established convention so we don't have mixed-up approaches. Or change all other test suites, but that is less than ideal.

But I do like the error test better than the current one, so if we choose to change, we could do it for those tests only.